### PR TITLE
Bug 1137886 — Log the number of participants for all POST /rooms/:token actions

### PR DESCRIPTION
**Pull request by Natim**
Based on #309.

https://bugzilla.mozilla.org/show_bug.cgi?id=1137886

### CODE REVIEW

File: loop/routes/rooms.js [hunk 1]
```diff
@@ -380,6 +381,10 @@ module.exports = function (apiRouter, conf, logError, storage, auth,
                           account: account
                         }, ttl, function(err) {
                           if (res.serverError(err)) return;
+
+                          // We've just added a new participant.
```
Comment (ametaireau) [inline loop/routes/rooms.js]: Avoid usage of "we" in the comments.

**Comment by ametaireau** [comment]
Can you add a test for this new behavior?

**Comment by Natim** [comment]
Yes sure.

**Comment by Natim** [comment]
Test added.

**Pull request merged**